tests/src/test_bitinputarchive.cpp

```diff
@@ -2029,4 +2030,65 @@ TEMPLATE_TEST_CASE( "BitInputArchive: Extracting a non-existing folder from an a
         REQUIRE_THROWS( info.extractFolderTo( testOutDir, folderPath ) );
         REQUIRE( fs::is_empty( testOutDir.path() ) );
     }
+}
+
+TEMPLATE_TEST_CASE( "BitInputArchive: Extract to callback",
```

Comment:
let me know:

- should I use multiple_items as a test? I see there are more items and those are nested, so I think it might be a better candidate for the test actually.